test/alternator/test_returnconsumedcapacity.py

```diff
@@ -190,3 +190,30 @@ def test_number_magnitude_key(test_table_sn):
             response = test_table_sn.put_item(Item={'p': p, 'c': num, 'a': x, 'val2': val2 + 'a'*10}, ReturnConsumedCapacity='TOTAL')
             assert 2 == response.get('ConsumedCapacity')["CapacityUnits"]
 
+# the simple delet item validate that when deleting a short item
```

Comment:
Please copy-edit this comment. It is full of English mistakes and typos.

Reply:
will do